src/ZipkinOpenTracing/ScopeManager.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace ZipkinOpenTracing;
+
+use OpenTracing\ScopeManager as OTScopeManager;
+use OpenTracing\Span as OTSpan;
+
+class ScopeManager implements OTScopeManager
+{
+    /**
+     * @var Scope
+     */
+    private $activeScope;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function activate(OTSpan $span)
+    {
+        $this->activeScope = new Scope($this, $span);
+
+        return $this->activeScope;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getActiveScope()
+    {
+        return $this->activeScope;
+    }
+
+    public function getScope(OTSpan $span)
+    {
+        throw new \RuntimeException('Not implemented');
```

Comment:
Are there plans for implementing this? In any case, if this is intended, it should not be `RuntimeException` but `BadFunctionCallException`.